install_config/monitoring/modules/determining_storage_requirements.adoc

```diff
@@ -0,0 +1,8 @@
+= Determining storage requirements
+
+How much storage you need depends on the number of pods. It is administrator's responsibility to dedicate sufficient storage to ensure that the disk does not become full.
```

Comment:
This might make more sense as a prereq for the assembly. (And you can xref from assembly to assembly.)